common/predictive-text/unit_tests/in_browser/CI.conf.js

```diff
@@ -1,172 +1,4 @@
-    //bs_firefox_win: {
-    //  os: 'Windows',
-    //  os_version: '10',
-    //  browser: 'firefox',
-    //  browser_version: '62.0'
-    //},
-    bs_chrome_win: {
-      os: 'Windows',
-      os_version: '10',
-      browser: 'chrome',
-      browser_version: '70.0'
-    },
-    // On recent versions of Edge, launcher fails to start and/or stop Edge successfully
-    //bs_edge_win: {
-    //  os: 'Windows',
-    //  os_version: '10',
-    //  browser: 'edge',
-    //  browser_version: '17.0'
-    //}
-  }
-
-  var CURRENT_ANDROID_LAUNCHERS = {
-    bs_native_android: {
-      os: 'android',
-      os_version: '7.1',
-      browser: 'firefox',
-      real_mobile: true,
-      device: 'Samsung Galaxy Note 8'
-    },
-    bs_chrome_android: {
-      os: 'android',
-      os_version: '7.1',
-      browser: 'chrome',
-      real_mobile: true,
-      device: 'Samsung Galaxy Note 8'
-    }
-  }
-
-  // Sadly, legacy IE isn't very testable with Mocha.  One of its dependencies requires a feature that is IE11+.
-
-  /*
-   * Final selection of the sets to be used for BrowserStack testing.
-   *
-   * Native JavaScript testing on iOS devices is not supported by BrowserStack at this time.
-   */
-  var FINAL_LAUNCHER_DEFS = mergeLaunchers( CURRENT_ANDROID_LAUNCHERS,
-                                            CURRENT_WIN_LAUNCHERS,
-                                            CURRENT_MAC_LAUNCHERS);
-
-  var FINAL_BROWSER_LIST = toBrowserList(FINAL_LAUNCHER_DEFS);
-
-  /*
-   * Final definition of our BrowserStack testing Karma configuration.
-   */
-
-  var specifics = {
-    // BrowserStack configuration options
-    browserStack: {
-      video: true,
-      browserDisconnectTimeout: 6e4, // 1 minute (60s => 60,000ms)
-      retryLimit: 3, // 0 is ignored.
-      startTunnel: true,
-    },
-
-    // Attempts to avoid generating a 'fail' exit code if one of our selected browsers on BrowserStack goes poof.
-    failOnEmptyTestSuite: false,
-
-    captureTimeout: 1.2e5, // in milliseconds
-    browserNoActivityTimeout: 6e4,
-    browserDisconnectTimeout: 6e4,
-    browserDisconnectTolerance: 3,
-
-    // Avoids generating a 'fail' exit code if one of our selected browsers on BrowserStack goes poof.
-    failOnEmptyTestSuite: false,
-
-    // test results reporter to use
-    // possible values: 'dots', 'progress'
-    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['teamcity', 'BrowserStack'],
-
-    // level of logging
-    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
-    logLevel: config.LOG_INFO,
-
-    // Concurrency level
-    // For CI, it really helps to keep a nice, clean set of output logs.
-    concurrency: 5,
-
-    customLaunchers: FINAL_LAUNCHER_DEFS,
-
-    browsers: FINAL_BROWSER_LIST
-  };
-
-  config.set(Object.assign(specifics, base));
-}
+module.exports = ci_config_adapter(BASE_CONFIG, "full");
```

Comment:
While our previous version of this config file aimed for browser coverage matching that of KMW itself, I think we can "get away" with a smaller set of test environments for this module.  There's less browser-specific behavior to worry about at its level.

```suggestion
module.exports = ci_config_adapter(BASE_CONFIG, "partial");
```

We can opt for a smaller test set with this change if desired.  Just thought I'd note this.